nebari-dev/nebari
# add authorized ip range variable for azure

## Reference Issues or PRs
Closes #2784 


## What does this implement/fix?

_Put a `x` in the boxes that apply_


## Testing


## How to test this PR?
Ensure you can deploy to azure with an existing azure config.

Then

Add a `authorized_ip_ranges` line to the azure provider of your nebari-config.yaml. Deploy to azure and ensure that access to api server is limited to that ip range.

example config block
```yaml
azure:
  authorized_ip_ranges: ["80.145.170.197/32"]  # make this your own ip address
  kubernetes_version: 1.29.2
  region: eastus
  storage_account_postfix: esrr
  node_groups:
    general:
      instance: Standard_D8_v3
      min_nodes: 1
      max_nodes: 1
    user:
      instance: Standard_D4_v3
      min_nodes: 0
      max_nodes: 5
    worker:
      instance: Standard_D4_v3
      min_nodes: 0
      max_nodes: 5
```



## Any other comments?
